src/app/features/dashboard-widgets/birthday-dashboard-widget/birthday-dashboard/birthday-dashboard.component.html

```diff
@@ -1,8 +1,10 @@
 <app-dashboard-list-widget
   icon="birthday-cake"
   theme="child"
-  subtitle="Upcoming Birthdays"
+  [subtitle]="subtitle ?? 'Upcoming Birthdays'"
   i18n-subtitle="Subtitle of the birthday widget"
+  [explanation]="explanation ?? 'List of children who have upcoming birthdays'"
```

Comment:
Let's not define an explanation here (sometimes the people may not be called "children", so it can become confusing; and I think the title is already self-explanatory)